### PR TITLE
DDF-2825: Move ConfiguratorSuite to DDF

### PR DESCRIPTION
**Pull request by tbatie**
#### What does this PR do?
The ConfiguratorSuite service is used to group the ConfiguratorFactory and Actions under one service.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@paouelle @peterhuffer 
#### Select relevant component teams: 
none
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@figliold 
@clockard 
#### How should this be tested? (List steps with links to updated documentation)
CI passes
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
For the general case, merging is gated on the following:

* Review by at least two contributors
* Review by at least two committers
* Successful CI
* Successful static analysis (SonarQube/Coverity/etc.)
* Heroing of change (details vary by ticket)

Certain exceptions may occur. For example, if there is a problem with the CI environment then a manually performed full build and static analysis check may be substituted _at the discretion of the repository maintainers_. Additionally, repository metadata such as this `PULL_REQUEST_TEMPLATE`, the repository `README`, and other non-functional items may be merged without requiring CI and static analysis.

Finally, an abbreviated review process may be in place for forward-port tickets immediately following branching and prior to branch divergence. In that case, approvals, CI, static analysis, and heroing that occur against the PR against the _release branch_ can be considered sufficient for reviewing the PR against the _master branch_.

Once the branches diverge, the process will return to normal.

Insofar as divergence is not a function of time but of change to the parent branch, the repository maintainers will determine when abbreviated review ends.

When abbreviated review is in effect, a note to that effect will be at the top of this template. Following is an example note:
```
##### ABBREVIATED REVIEW BETWEEN 2.12.X AND MASTER IS IN EFFECT
____

```

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by tbatie** [comment]
build now

File: platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.internal.admin.configurator.actions;
+
+import org.codice.ddf.admin.configurator.ConfiguratorFactory;
+
+/** Provides access to a ConfiguratorFactory and a collection Actions */
+public interface ConfiguratorSuite {
```
Comment (clockard) [inline platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java]: Should probably add the experimental header to this interface

File: platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  ConfiguratorFactory getConfiguratorFactory();
```
Comment (paouelle) [inline platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java]: ❓ Should we not document public methods like these?

File: platform/admin/configurator/admin-configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfiguratorSuiteImpl.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import org.codice.ddf.admin.configurator.ConfiguratorFactory;
+import org.codice.ddf.internal.admin.configurator.actions.BundleActions;
+import org.codice.ddf.internal.admin.configurator.actions.ConfiguratorSuite;
+import org.codice.ddf.internal.admin.configurator.actions.FeatureActions;
+import org.codice.ddf.internal.admin.configurator.actions.ManagedServiceActions;
+import org.codice.ddf.internal.admin.configurator.actions.PropertyActions;
+import org.codice.ddf.internal.admin.configurator.actions.ServiceActions;
+import org.codice.ddf.internal.admin.configurator.actions.ServiceReader;
+
+public class ConfiguratorSuiteImpl implements ConfiguratorSuite {
```
Comment (paouelle) [inline platform/admin/configurator/admin-configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfiguratorSuiteImpl.java]: ✏️  Maybe a class description

File: platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java (hunk 2 above)
Comment (tbatie) [reply platform/admin/configurator/admin-configurator-actions-api/src/main/java/org/codice/ddf/internal/admin/configurator/actions/ConfiguratorSuite.java]: I figured that was covered by the interface description and the method return type.

**Comment by tbatie** [comment]
build now

**Comment by tbatie** [comment]
build now

**Pull request merged**